mne_bids/write.py

```diff
@@ -1327,15 +1353,23 @@ def write_raw_bids(raw, bids_path, events_data=None,
     if not convert:
         convert = ext not in ALLOWED_DATATYPE_EXTENSIONS[bids_path.datatype]
 
-    # check if there is an BIDS-unsupported MEG format
-    if bids_path.datatype == 'meg' and convert and not anonymize:
+        if convert and symlink:
+            raise RuntimeError(
+                'The input file format is not supported by the BIDS standard. '
+                'To store your data, MNE-BIDS would have to convert it. '
+                'However, this is not possible since you set symlink=True. '
+                'Deactiave symbolic links by passing symlink=False to allow '
```

Comment:
```suggestion
                'Deactivate symbolic links by passing symlink=False to allow '
```

Reply:
Looks great @hoechenberger!
About the split files: is this something that could happen directly at storage by the acquisition software, or are those only created by mne functions? (Just wondering if we should instruct people to calculate file size beforehand to prevent this).